mozilla_bitbar_devicepool/configuration.py

```diff
@@ -56,12 +56,30 @@ def __init__(self, message):
 class ConfigurationFileException(ConfigurationException):
     pass
 
+class ConfigurationFileDuplicateFilenamesException(ConfigurationException):
```

Comment:
Perhaps inherit from ConfigurationFileException instead so that ConfigurationFileDuplicateFilenamesException is a child of ConfigurationFileException ?

Reply:
Done.